__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const app = require("../app");
+const db = require("../db/connection");
+const request = require("supertest");
+const testData = require("../db/data/test-data/index.js");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/topics", () => {
+  test("GET - 200: responds with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        const { topics } = res.body;
+        expect(topics.length).toBe(3);
```

Comment:
This is a fragile test and will give a false negative if a new topic is ever added. Consider how you can test that the endpoint responds appropriately for an array of any length greater than 0.